.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
One thing is at an org level I had to change it to have codespaces be billed by Bodo, so we may need to restrict access on the open source Bodo repo.